tests/utils/test_dag_processing.py

```diff
@@ -85,6 +87,23 @@ class TestDagFileProcessorManager(unittest.TestCase):
     def setUp(self):
         clear_db_runs()
 
+    def run_processor_manager_one_loop(self, manager, parent_pipe):
+        if not manager._async_mode:
+            parent_pipe.send(DagParsingSignal.AGENT_RUN_ONCE)
+
+        results = []
+
+        while True:  # pylint: disable=too-many-nested-blocks
+            manager._run_parsing_loop()
+
+            while parent_pipe.poll(timeout=0.01):
+                obj = parent_pipe.recv()
+                if isinstance(obj, DagParsingStat):
+                    if obj.done:
+                        return results
+                    continue
```

Comment:
```suggestion
                if isinstance(obj, DagParsingStat) and obj.done:
                    return results
```
This should please pylint